content/releases/3.breaking-changes/2.version-11.md

```diff
@@ -3,6 +3,32 @@ title: Version 11
 description: Breaking changes may require action on your part before upgrading.
 ---
 
+## Version 11.10.0
+
+#### Typed services for API extensions using TypeScript
+
+The services exposed to API extensions using TypeScript are now fully typed instead of `any` this can potentially cause new type errors when building extensions.
+
+Arguments of service methods are now strictly typed which can result in type errors for broader types that would not error before:
+- The ItemsService constructor now expects the collection name to be a `string` so this will error on `string | undefined` (or other unions).
+- Similarly functions like `service.readOne()`/`service.readMany()` now expects `string | number` for their primary keys and will error for nullable types
```

Comment:
```suggestion
The services exposed to API extensions using TypeScript are now fully typed instead of `any`, this may cause new type errors when building extensions.

Arguments of service methods are now strictly typed which can result in type errors for broader types that would not error before:
- The ItemsService constructor now expects the collection name to be a `string` and will error on `string | undefined` (or other unions).
- Similarly functions like `service.readOne()`/`service.readMany()` now expect `string | number` for their primary keys and will error for nullable types
```